conf/deployment/fusion_aas/managed_3az_consumer_dev_3m_3w_m52x.yaml

```diff
@@ -6,6 +6,7 @@ ENV_DATA:
   # or via rosa_service i.e appliance mode
   cluster_type: 'consumer'
   cluster_namespace: 'fusion-storage'
+  service_namespace: 'managed-fusion'
```

Comment:
We have a lot more provider and consumer config files (for 1 az vs 3 az, dev vs qe vs stable, m52 vs 54). Do you think it would be better to include service_namespace in all those files or maybe just in https://github.com/red-hat-storage/ocs-ci/blob/master/conf/deployment/rosa/consumer_cluster_type.yaml and https://github.com/red-hat-storage/ocs-ci/blob/master/conf/deployment/rosa/provider_cluster_type.yaml?

Reply:
I think that it will need to be part of all fusion config files to differentiate from other platforms.